packages/expo-notifications/src/__tests__/Notifications-test.ts

```diff
@@ -123,6 +123,47 @@ it(`verifies weekly trigger input validation`, async () => {
   }
 });
 
+it(`verifies monthly trigger handling`, async () => {
+  const trigger: NotificationTriggerInput = {
+    type: SchedulableTriggerInputTypes.MONTHLY,
+    day: 5,
+    hour: 12,
+    minute: 30,
+  };
+  const input = {
+    ...notificationTriggerInputTest,
+    trigger,
+  };
+  await scheduleNotificationAsync(input);
+  expect(NotificationScheduler.scheduleNotificationAsync).toHaveBeenLastCalledWith(
+    input.identifier,
+    input.content,
+    {
+      ...input.trigger,
+    }
+  );
+});
+
+it(`verifies monthly trigger input validation`, async () => {
+  const trigger: NotificationTriggerInput = {
+    type: SchedulableTriggerInputTypes.MONTHLY,
+    day: 32,
+    hour: 12,
+    minute: 30,
+  };
+  const input = {
+    ...notificationTriggerInputTest,
+    trigger,
+  };
+  try {
+    await scheduleNotificationAsync(input);
```

Comment:
we expect this to throw. However, if it doesn't throw, the test will pass when instead it should fail.

Jest has this https://jestjs.io/docs/expect#expecthasassertions / https://jestjs.io/docs/expect#expectassertionsnumber

But we probably don't have these here. Probably we should throw some error after `await scheduleNotificationAsync(input);` which will be caught by the catch block but the expectations will fail.

At least one more test case here makes the same assumption.

Reply:
Good point! I'll go through all these tests to make sure that all the tests for exceptions will fail if they do not throw.

Reply:
Fixed!